components/library/jxrlib/Makefile

```diff
@@ -19,9 +19,10 @@ include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		jxrlib
 COMPONENT_VERSION=	0.2.4
+COMPONENT_REVISION=	1
 COMPONENT_SUMMARY=	Decode and encode JPEG-XR data from Java
 COMPONENT_PROJECT_URL=	https://github.com/glencoesoftware/jxrlib
-COMPONENT_FMRI=		library/jxlib
+COMPONENT_FMRI=		library/jxrlib
 COMPONENT_CLASSIFICATION=System/Libraries
```

Comment:
When you change COMPONENT_FMRI you'll need to obsolete the old package. You can do that with a local history file. In this case you should use the special syntax to forward from the old to the new fmri. In this case you also don't need to set the COMPONENT_REVISION because you'll create a new package.